tests/integration/suite/test_rbac.py

```diff
@@ -396,18 +397,23 @@ def test_appropriate_users_can_see_kontainer_drivers(user_factory):
     assert len(kds) == 0
 
 
-def test_readonly_cannot_perform_app_action(admin_mc, admin_pc, user_factory,
```

Comment:
Should we just rip out user_factory all together? Looks like it is the source of these errors.